neofetch

```diff
@@ -2183,12 +2183,25 @@ END
         ;;
 
         "urxvt" | "urxvtd" | "rxvt-unicode" | "xterm")
-            term_font="$(grep -i "${term/d}\**\.*font" < <(xrdb -query))"
+            xrdb="$(xrdb -query)"
+            term_font="$(grep -i "${term/d}\**\.*font" <<< "$xrdb")"
             term_font="${term_font/*"*font:"}"
             term_font="${term_font/*".font:"}"
             term_font="${term_font/*"*.font:"}"
             term_font="$(trim "$term_font")"
 
+            if [[ -z "$term_font" && "$term" == "xterm" ]]; then
+                term_font="$(grep -E '^XTerm.vt100.faceName' <<< "$xrdb")"
+                term_font="${term_font/*"faceName:"}"
+                term_font="$(trim "$term_font")"
```

Comment:
`trim` is probably unneeded. Every string gets trimmed before printing anyway. Unless you've got a reason for keeping it?